### DISCUSSION TITLE
如何修改封面中的"论文题目"

### DISCUSSION THREAD

**Discussion by SunSwallow** [sjtug/SJTUThesis#756] [提问] [answered]
想用模板写课程报告,感觉封面中题目前的"论文题目"不太合适,请问如何"自定义修改"论文题目",例如改成"报告题目"?

**Comment by skyzh** [comment, marked as answer]
这个在模版里是写死的，只能直接改模版源文件了：

https://github.com/sjtug/SJTUThesis/blob/90ce4a3d5fb729143bc57c8e2db021417a985f90/texmf/tex/latex/sjtuthesis/sjtuthesis-undergraduate.ltx#L137

**Reply by SunSwallow** [reply]
谢谢~

**Comment by AAAzuree** [comment]
请问更新后的模板该在哪里修改呀

**Reply by LogCreative** [reply]
新版模板应该不存在“论文标题”这种字样？

**Reply by AAAzuree** [reply]
新版论文最上面的校徽下面有“xx学位论文”的字样，我看了下源码格式应该是“degree+学位论文”？我只找到了如何修改“学位论文”的源代码，如果不是学士/硕士/博士论文的话该如何不填写呢x

**Reply by LogCreative** [reply]
可以直接在导言区添加:
```latex
\sjtusetup{
  info/zh/subject={XX论文},
  info/en/subject={Thesis XX},
}
```
